.gitignore

```diff
@@ -35,3 +35,6 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+# caching data generated when hosted on Vercel
+/files
```

Comment:
Newlines at the end of files VS Code Extension (Editorconfig)

Reply:
This is odd, as this is indeed a setting in the `.editorconfig` - not sure why this didn't get automatically added.